README.md

```diff
@@ -1,15 +1,59 @@
+[![Open in Visual Studio Code](https://classroom.github.com/assets/open-in-vscode-718a45dd9cf7e7f842a935f5ebbe5719a5e09af4491e668f4dbf3b35d5cca122.svg)](https://classroom.github.com/online_ide?assignment_repo_id=12577629&assignment_repo_type=AssignmentRepo)
 # Theory vs. Practice
 
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
 
+  - Ignoring Lower order Terms: Asymtotic analysis focuses on the dominant term as the input sizes increases.
+    Example - if an algerith has a time complexity of $\ 2n^2 + 3n + 1$
+
+  - Algorithms may have different behaviors for best-case, average-case, and worst-case scenarios. Asymptotic analysis often focuses on the worst-case scenario, but in practice, the distribution of input data may lead to different average-case behavior.
```

Comment:
This sounds like it would only happen if you mis-identify the case. Not an issue with asymptotic analysis as such.